include/grass/gis.h

```diff
@@ -129,6 +129,13 @@ static const char *GRASS_copyright __attribute__ ((unused))
 #define CONFIG_DIR ".grass8"
 #endif
 
+#ifndef MAX
```

Comment:
Why use a previously defined MAX or MIN? You don't know what these previously defined MAX and MIN macros are doing, thus for protection add
```
#ifdef MAX
#undef MAX
#endif
``` 
for MIN accordingly.

Reply:
Yes, that's a valid point. libgeotiff before commit 9b9ea38 (the one in Ubuntu 20.04) defines these incorrectly, for instance.